platform/ui/src/components/measurementTable/MeasurementTableItem.js

```diff
@@ -1,6 +1,6 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-import { withTranslation } from '../../utils/LanguageProvider';
+import { withTranslation } from '../../utils/providers';
```

Comment:
Is it worth moving all `providers` to a root folder named `contextProviders` instead of jamming them into `utils`?

Reply:
Beyond simplifying all imports I think a new folder just keep things more organized. Do you think its not worth it?

Reply:
Oh, I was agreeing that they should have their own place to live. I'm just not sure if under `utils` makes sense. I wouldn't think of a ContextProvider as a "utility".

I could see them either:

- Living next to the component they provide the context for
- or living under `src/contextProviders`, as a quick way to see all of the different context providers for our UI library

Reply:
> Oh, I was agreeing that they should have their own place to live. I'm just not sure if under `utils` makes sense. I wouldn't think of a ContextProvider as a "utility".
> 
> I could see them either:
> 
> * Living next to the component they provide the context for
> * or living under `src/contextProviders`, as a quick way to see all of the different context providers for our UI library

oh! yeah, I was wondering why they were in utils as well. Ill move them fore sure! thx.

Reply:
Done, thx.